posixlib/src/main/scala/scala/scalanative/posix/sys/wait.scala

```diff
@@ -0,0 +1,43 @@
+package scala.scalanative.posix.sys
+
```

Comment:
In posixlib implementations I find it useful to leave a one line comment
giving the Posix version (year, revision level) and URL. I think that you
are using https://pubs.opengroup.org/onlinepubs/9699919799/ here
(which is goodness).   Some devo down the line will bless you.

I believe that you are also implementing the XSI functions.  If, like here,
you are intending to implement all XSI, I would say so.   That helps
reviewers and readers twig to unintended missing XSI methods.

I think that in other files where only one or two XSI functions were implemented (for either historical or unfortunate reasons), I have put
a one line comment // XSI before the function.  I have done this as I 
have visited or changed files and have been inconsistent at best.

Posix has changed over the years and various SN posixlib files have been implemented to various Posix standards and to various degrees of completion relative to that standard.  Understandably, people implemented what they needed; completeness could come later.

With regards to wait.h, the issue of "which version" comes to the front because, by my reading, some requirements, such as including "types.h" 
have been __removed__.  Any reviewer following an old but easily found on the Web spec (or an old memory of a spec learned by heart) may wonder why, say, "types.h" (and I think some of the signal.h") parts are missing.

Reply:
done